<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdFreshStartDialog</class>
 <widget class="QDialog" name="QxrdFreshStartDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qxrd Startup Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Detector Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="m_RunSpecServer">
     <property name="text">
      <string>Run 'spec' server socket</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>On Port:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QSpinBox" name="m_SpecServerPort">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="m_RunSimpleServer">
     <property name="text">
      <string>Run 'simple' server socket</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>On Port:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QSpinBox" name="m_SimpleServerPort">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QCheckBox" name="m_DefaultLayout">
     <property name="text">
      <string>Reset Screen Layout to Defaults</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Current log file:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="m_CurrentLogfileBrowse">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QGroupBox" name="m_DebugWidgets">
     <property name="title">
      <string>Debug Settings</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="6">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>458</width>
       <height>146</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>319</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="4" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QLineEdit" name="m_CurrentLogFile"/>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QComboBox" name="m_DetectorTypeCombo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QxrdFreshStartDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QxrdFreshStartDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
